src/Hng.Web/appsettings.json

```diff
@@ -7,22 +7,22 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DefaultConnectionString": ""
+    "DefaultConnectionString": "Server=localhost;Port=5432;Database=csharp_dev;User Id=csharp_superuser;Password=421I7eoZzmRm1KOCDOW3hQ;"
```

Comment:
remove your personal details. you can have a separate appsettings.Development.json file for this. add to gitignore. but leave the appsettings file as it is.